src/plugins/intel_gna/legacy/tests/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if(CMAKE_COMPILER_IS_GNUCXX)
+    ie_add_compiler_flags(-Wno-missing-declarations)
+    ie_add_compiler_flags(-Wno-bool-compare)
+endif()
+
+set(TARGET_NAME ov_legacy_transformations_tests)
```

Comment:
What if we merge these transformation tests with tests on GNA plugin?

Reply:
I though about it, but anyway we still have deprecated GnaFunctionalTests and InferenceEngineUnitTests, which I don't want merge into a single binary. Taking into account the life time of GNA plugin, let's keep it as is